src/worker.cc

```diff
@@ -448,6 +451,23 @@ void parse_locations(const rapidjson::Document& doc,
       midgard::logging::Log(node + "_count::" + std::to_string(request_locations->Size()),
                             " [ANALYTICS] ");
     }
+
+    // push the date time information down into the locations
+    if (!had_date_time && options.has_date_time() && locations->size()) {
```

Comment:
!locations->empty()

Reply:
this is pbf generated code, pbf did not generate `empty` for some older versions so to maintain compatibility we use `size`